linux-bore/linux-cachyos-pds.install

```diff
@@ -0,0 +1,16 @@
+post_install() {
+    cat << EOF
+You install the linux-cachyos kernel variant with the PDS/BMQ scheduler.
+Please note that this scheduler can cause various performance and stability
+issues (especially on AMD Ryzen CPUs) and we don't provide official
+support for fixing them. If you encounter any problems while using the kernel,
+please make sure that that it is not a PDS-specific problem before reporting it
```

Comment:
Also here we should change it to "not a PRJC-specific problem"

Reply:
I think it is better to use PDS for simplicity, because the PRJC project name is not always obvious to ordinary users. As I see it, "not a BMQ or PDS specific problem" is more suitable

Reply:
Yes, it is better than just PDS.

Reply:
Done.